app/client/ui/ShareMenu.ts

```diff
@@ -270,6 +279,23 @@ function menuExports(doc: Document, pageModel: DocPageModel) {
   const gristDoc = pageModel.gristDoc.get();
   if (!gristDoc) { return null; }
 
+  const onClick = dom.on('click', e => {
+    const currentPage = pageModel.gristDoc.get()?.activeViewId.get();
+    const notDataPage = typeof currentPage !== 'number';
+    if (notDataPage) {
+      // Disable navigation.
+      e.preventDefault();
+      // Show warning.
+      setTimeout(() => reportError(new UserError(
+        t("Exporting is only available from data pages. Please select a data page and try again."),
```

Comment:
Is "data page" clear enough to the user? Maybe "document page"? I am not sure if we've been consistent about nomenclature, but for example, the raw data page seems to me like a "data page", but the warning shows up on that page too.

Reply:
Done